src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -45,6 +49,9 @@ public Shooter(ShooterIO shooterIO) {
   @Override
   public void periodic() {
     shooterIO.updateInputs(shooterIOAutoLogged);
+    if (Constants.FeatureFlags.kTuningMode) {
+      shooterIO.setShooterVelocity(shooterMotorVelocityInput.get());
```

Comment:
You need another tunable number for the follower motor since they run at different speeds sometimes.